osu.Game/Beatmaps/IWorkingBeatmap.cs

```diff
@@ -44,7 +44,7 @@ public interface IWorkingBeatmap
         /// <summary>
         /// Retrieves the <see cref="Skin"/> which this <see cref="WorkingBeatmap"/> provides.
```

Comment:
xmldoc should be adjusted to match
```suggestion
        /// Retrieves the <see cref="IBeatmapSkin"/> which this <see cref="WorkingBeatmap"/> provides.
```